guides/simulcast.md

```diff
@@ -0,0 +1,68 @@
+# Simulcast
+
+Simulcast is a technique where a client sends multiple encodings of the same video to the server and the server is responsbile for choosing and forwarding proper encoding to proper receiver (other client). The encoding selection is dynamic (i.e. SFU switches between encodings in time) and it is based on:
+
+* receiver awailable bandwidth
+* receiver preferences (e.g. explicit request to receive video in HD resolution instead of FHD)
+* UI layaout (e.g. videos being displayed in smaller video tiles will be sent in lower resolution)
+
+At the moment, Membrane supports only receiver preferences i.e. receiver can chose which encoding it is willing to receive. Additionaly, sender can turn off/on specific encoding. Membrane RTC Engine will detect changes and switch to another available encoding.Simulcast is a technique where client sends multiple encodings (different resolutions) of the same
+video to a server and the server forwards proper encoding to each other client basing on 
+client preferences, network bandwidth or UI layaout.
+
+## Turning simulcast on/off
+
+On the client side simulcast can be enabled while adding new track e.g.:
+
+```ts
+    // create MembraneWebRTC class instance
+    // ...
+    // add simulcasted track
+    let trackId = webrtc.addTrack(track, stream, {}, true);
+```
+
+This will add a new track that will be sent in three versions:
+* original (identified as `h`)
+* original scaled down by 2 (identified as `m`)
+* original scaled down by 4 (identified as `l`)
+
+Those settings are not configurable at the moment.
+
+> #### Minimal required resolution {: .warning}
+>
+> To make all encodings working, original track resolution has to be at least 1280x720.
+> In other case, browser might not be able to scale resolution down.
+
+On the server side, simulcast is by default enabled and cannot be turned off at the moment.
```

Comment:
Does it mean that we can not remove simulcast track?

Reply:
Nope, it just mean that whenever client adds track as a simulcast one, server will always accept it.

Reply:
I want to make sure, when the server refuses the simulcast track it would change the simulcast track to the non-simulcast track in SDP Answer?